p/scripts/main.js

```diff
@@ -1068,6 +1078,15 @@ function init_stream(stream) {
 		}
 	};
 
+	if (!navigator.share) {
+		// https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share
+		// do not show the menu entry if browser does not support navigator.share
+		document.styleSheets[0].insertRule(
```

Comment:
Ah, I overlooked that this might not work until enough of the DOM is loaded, otherwise errors sometimes happen depending on the exact orders
![image](https://user-images.githubusercontent.com/1008324/159922423-b5580b47-3bdf-40b2-a283-81fff2b06c74.png)
I believe this code should be moved to a function called by `init_afterDOM()`.

Reply:
https://github.com/FreshRSS/FreshRSS/pull/4321